### PR TITLE
CORE-11758-merge development into release/corda-5-beta-2 release branch

### PR DESCRIPTION
**Pull request by mattbradburyr3**
updating the release branch for RC02 changes

### CODE REVIEW

File: buildSrc/src/main/java/com/r3/csde/VNode.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.r3.csde;
+
+
+/**
+ * This class is a representation of a Vnode used to express the vNodes required on the network.
+ */
+
+public class VNode{
```
Comment (yarivus) [inline buildSrc/src/main/java/com/r3/csde/VNode.java]:
```suggestion
public class VNode {
```

File: buildSrc/src/main/java/com/r3/csde/dtos/CpiIdentifierDTO.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+public class CpiIdentifierDTO {
+
+    // Note, these DTOs don't cover all returned values, just the ones required for CSDE
+    private String cpiName;
+    private String cpiVersion;
+    private String signerSummaryHash;
+
+    public CpiIdentifierDTO() {
+    }
```
Comment (yarivus) [inline buildSrc/src/main/java/com/r3/csde/dtos/CpiIdentifierDTO.java]:
```suggestion
    public CpiIdentifierDTO() { }
```

File: contracts/src/main/kotlin/com/r3/developers/csdetemplate/utxoexample/states/ChatState.kt [hunk 3]
```diff
@@ -24,7 +24,11 @@ data class ChatState(
     // The message
     val message: String,
     // The participants to the chat, represented by their public key.
-    override val participants: List<PublicKey>) : ContractState {
+    private val participants: List<PublicKey>) : ContractState {
+
+    override fun getParticipants(): List<PublicKey> {
+        return participants
+    }
```
Comment (yarivus) [inline contracts/src/main/kotlin/com/r3/developers/csdetemplate/utxoexample/states/ChatState.kt:31]:
```suggestion
    override fun getParticipants(): List<PublicKey> { return participants }
```

**Review by yarivus — APPROVED** [review]
LGTM

File: buildSrc/src/main/java/com/r3/csde/VNode.java (hunk 1 above)
Comment (mattbradburyr3) [reply buildSrc/src/main/java/com/r3/csde/VNode.java]: Done

File: buildSrc/src/main/java/com/r3/csde/dtos/CpiIdentifierDTO.java (hunk 2 above)
Comment (mattbradburyr3) [reply buildSrc/src/main/java/com/r3/csde/dtos/CpiIdentifierDTO.java]: Done

**Pull request merged**